AntonEmery/cards4-template.js

```diff
@@ -0,0 +1,141 @@
+var makeCard = // receive factory with external name `makeCard`
```

Comment:
Solid work! It looks like all your code is producing the expected results; nice job.